libs/ui/core/src/lib/components/generic-form-cancel-button/generic-form-cancel-button.tsx

```diff
@@ -3,12 +3,11 @@ import { Button } from '@material-ui/core';
 import { useTranslation } from 'react-i18next';
 import { Add } from '@material-ui/icons';
 import { useStyles } from './generic-form-cancel-button.styles';
-
+import { variables } from '@energyweb/zero-theme';
 /* eslint-disable-next-line */
```

Comment:
I think we can remove this disabling comment

Reply:
sure, will remove, just decided not to touch something that is not my change